src/contact/4C_contact_input.hpp

```diff
@@ -32,10 +32,10 @@ namespace CONTACT
 
   /// Type of contact adhesion law
   /// (this enum represents the input file parameter ADHESION)
-  enum AdhesionType
+  enum class AdhesionType
   {
-    adhesion_none,  ///< no adhesion
-    adhesion_bound  ///< fix bounded adhesion
+    none,  ///< no adhesion
+    bound  ///< fix bounded adhesion
```

Comment:
```suggestion
    bounded  ///< fix bounded adhesion
```

To increase the symmetry between enum and input name? I will run a script in the context of #463 to automatically replace such `string_to_integral_parameter` calls with the new equivalents. If the string options exactly match the enum names, I can directly replace with `paramter<Enum>` which is the best case. Otherwise, I will use `deprecated_selection`. So, since you clean this anyway, do you think you can drop the `None` spelling and the `b` options as well or would this break many files?

Reply:
Done.